### ISSUE TITLE
Bob can't move when surrounded by objects

### ISSUE THREAD

**Issue by patrickcarlohickman** [darian311/VillageMakeover#18]
Below is a screenshot of how my game starts. Bob won't walk anywhere no matter where I click. I assume it is because he is surrounded by objects.

![image](https://cloud.githubusercontent.com/assets/6036266/6818499/318cddc4-d287-11e4-8cc9-bb169be4bf0a.png)

**Comment by darianhickman** [comment]
Yeah,  it's definitely bug.  Our solution is to make Bob walk over top
objects.

On Wed, Mar 25, 2015 at 12:41 AM, Patrick Carlo-Hickman <
notifications@github.com> wrote:

> Below is a screenshot of how my game starts. Bob won't walk anywhere no
> matter where I click. I assume it is because he is surrounded by objects.
> 
> [image: image]
> https://cloud.githubusercontent.com/assets/6036266/6818499/318cddc4-d287-11e4-8cc9-bb169be4bf0a.png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/darian311/VillageMakeover/issues/18.

## 

Darian Hickman |  (310) 439-8573  | http://www.linkedin.com/in/darianhickman
